Packs/Phishing/README.md

```diff
@@ -1,24 +1,27 @@
-- Run checks for SSL certificates of URLs, email address breach involvement, domain-squatting and email authenticity using SPF, DKIM and DMARC checks.
-- Identify similar phishing incidents belonging to the same campaign, provide visibility and manual or automatic actions to respond to such incidents.
-- Calculate severity for the incident based on initial severity provided, indicator reputations, email authenticity check and critical assets if any are involved.
-- Remediate the incident by blocking malicious indicators, searching and deleting malicious emails upon analyst approval
-- Engage with the end user regarding the incident such as notifying them of receipt of email and providing further instructions if email is found to be malicious.
+- Retrieves emails from user inboxes or ingests them using mail listeners.
+- Creates a phishing incident within Cortex XSOAR associated with the email.
+- Extracts and enriches all indicators from email attachments. 
+- Analyzes files and provides reputation using your sandbox and threat intelligence integrations.
+- Generates a screenshot of the email and embedded links, and calculates reputation for all indicators involved.
+- Runs checks for SSL certificates of URLs, email address breach involvement, domain-squatting and email authenticity using SPF, DKIM and DMARC checks.
+- Identifies similar phishing incidents belonging to the same campaign, providing visibility and manual or automatic actions to respond to such incidents.
+- Calculates severity for the incident based on the provided initial severity, indicator reputations, email authenticity check, and critical assets if any are involved.
+- Remediates the incident by blocking malicious indicators, searching for and deleting malicious emails upon analyst approval.
+- Engages with the end user regarding the incident such as notifying them of receipt of email and providing further instructions if an email is found to be malicious.
 
 As part of this pack, you will also get out-of-the-box phishing incident views, a full layout and automation scripts. All of these are easily customizable to suit the needs of your organization.
 
 _For more information, visit our [Cortex XSOAR Developer Docs](https://xsoar.pan.dev/docs/reference/playbooks/phishing-investigation---generic-v2)._
 
 _If you're new to our Phishing Content Pack, check out our [Setting Up a Phishing Incident in Cortex XSOAR](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-2/cortex-xsoar-tutorials/tutorials/phishing-use-case.html) tutorial._
 
-![Phishing_Investigation_Generic_v2](https://raw.githubusercontent.com/demisto/content/952153821f8de373a5d527df653bfa845d32f77e/Packs/Phishing/doc_files/Phishing_Investigation_-_Generic_v2_-_6_0.png)
+![Phishing_Investigation_Generic_v2](https://raw.githubusercontent.com/demisto/content/13bc88b052c5039775cd0f569259e7fd0e1b7dea/Packs/Phishing/doc_files/Phishing_-_Generic_v3.png)

```

Comment:
Change to relative path

Reply:
I get an error in the CI validations process when using a relative path in the pack README file.